errai-bus/src/main/java/org/jboss/errai/bus/server/servlet/CSRFTokenFilter.java

```diff
@@ -74,19 +73,24 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
       final HttpSession session = httpRequest.getSession(false);
 
       final byte[] bytes;
+      final String output;
       final String responseContentType = responseWrapper.getContentType();
       if (session != null && responseContentType != null && responseContentType.toLowerCase().startsWith("text/html")) {
+
         CSRFTokenCheck.INSTANCE.prepareSession(session, log);
         final Document document = Jsoup.parse(responseWrapper.toString());
-        document.head().prepend("<script>var " + ERRAI_CSRF_TOKEN_VAR + " = '" + CSRFTokenCheck.getToken(session) + "';</script>");
-        bytes = document.html().getBytes("UTF-8");
+        String injectedScript = "<script>var " + ERRAI_CSRF_TOKEN_VAR + " = '" + CSRFTokenCheck.getToken(session) + "';</script>";
+        document.head().prepend(injectedScript);
+        output = document.html();
+        bytes = output.getBytes("UTF-8");
```

Comment:
Nitpick: The line `bytes = output.getBytes("UTF-8")` is identical in both if and else branches. It should be moved after the if-else.

Reply:
ok